pipeline/frontend/yaml/compiler/convert.go

```diff
@@ -68,7 +68,7 @@ func (c *Compiler) createProcess(name string, container *yaml.Container, section
 	}
 
 	if !detached || len(container.Commands) != 0 {
-		workingdir = path.Join(c.base, c.path)
+		workingdir = path.Join(c.base, c.path, container.Directory)
```

Comment:
can we move this out to it's own function?

Reply:
just in the same code-file for now but that will make it easyer to maintain the code